dev-qt/prison/prison-5.41.0.recipe

```diff
@@ -0,0 +1,71 @@
+SUMMARY="A Qt API for generating QRCode & DataMatrix barcodes"
+DESCRIPTION="prison is a Qt-based barcode abstraction layer/library that \
+provides uniform access to generation of barcodes with data. The API supports \
+two types of barcodes: QRCode and DataMatrix."
+HOMEPAGE="https://api.kde.org/frameworks/prison/html/index.html"
+COPYRIGHT="2010-2017 KDE Foundations"
+LICENSE="GNU LGPL v3"
+REVISION="1"
+SOURCE_URI="https://download.kde.org/stable/frameworks/${portVersion%.*}/prison-$portVersion.tar.xz"
+CHECKSUM_SHA256="b1ef221964d479255ac1ded2a96b73f90723b8e69f3b3c9a86dd1f02860a7960"
+
+ARCHITECTURES="x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion=$portVersion
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	prison$secondaryArchSuffix = $portVersion
+	lib:libkf5prison$secondaryArchSuffix = $portVersion compat >= 5
```

Comment:
Oops, you missed the small change at the end of this line:
```diff
-	lib:libkf5prison$secondaryArchSuffix = $portVersion compat >= 5
+	lib:libkf5prison$secondaryArchSuffix = $libVersionCompat
```
Everything else is OK :)